### DISCUSSION TITLE
Can’t display image from lfs

### DISCUSSION THREAD

**Discussion by t1anz0ng** [community/community#28168] [Mobile]
For example, README Preview image can't display: https://github.com/t1anz0ng/iftree

![image](https://user-images.githubusercontent.com/3898642/182598225-8b05aa3b-4a6a-4916-9400-a278e0d80c6f.png)

<details>

<summary>ℹ️ Additional Debug Info</summary>
GitHub for iOS, Version 1.74.0 (74357630)
Device: iPhone 12 (iOS 15.5)
</details>

**Comment by stevepopovich** [comment]
Hi @t1anz0ng. I see the issue. You believe the issue is due to those files being in Git LFS?

**Reply by t1anz0ng** [reply]
> Hi @t1anz0ng. I see the issue. You believe the issue is due to those files being in Git LFS?

Hi @stevepopovich ! 
I think so, since I was storing image straight in the repo and It worked fine. 

Plus, It works in browser on both computers and mobile now.

**Reply by stevepopovich** [reply]
It works on mobile web, but not native mobile correct? That is what I am seeing too. I will file a bug!

**Reply by t1anz0ng** [reply]
> It works on mobile web, but not native mobile correct? That is what I am seeing too. I will file a bug!

yeah you’re right, thanks!
